Cannot build kinetic-devel with catkin_make

```
-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-- ~~  traversing 8 packages in topological order:
-- ~~  - industrial_core (metapackage)
WARNING: The CMakeLists.txt of the metapackage 'industrial_core' contains non standard content. Use the content of the following file instead: /home/victor/catkin_test/build/catkin_generated/metapackages/industrial_core/CMakeLists.txt
-- ~~  - industrial_deprecated
-- ~~  - industrial_msgs
-- ~~  - simple_message
-- ~~  - industrial_utils
-- ~~  - industrial_robot_client
-- ~~  - industrial_robot_simulator
-- ~~  - industrial_trajectory_filters
-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
CMake Error at /opt/ros/kinetic/share/catkin/cmake/catkin_workspace.cmake:95 (message):
  This workspace contains non-catkin packages in it, and catkin cannot build
  a non-homogeneous workspace without isolation.  Try the
  'catkin_make_isolated' command instead.
Call Stack (most recent call first):
  CMakeLists.txt:63 (catkin_workspace)


-- Configuring incomplete, errors occurred!
See also "/home/victor/catkin_test/build/CMakeFiles/CMakeOutput.log".
See also "/home/victor/catkin_test/build/CMakeFiles/CMakeError.log".
Invoking "cmake" failed

```

--- a/industrial_core/CMakeLists.txt
+++ b/industrial_core/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 2.8.3)
 project(industrial_core)
 find_package(catkin REQUIRED)
 catkin_metapackage()

